src/resources/spinners.js

```diff
@@ -28,10 +28,10 @@ export default [
       <div></div>
     </div>
     `,
-    author: 'RickyFortNoSeMurio'
+    css: `@keyframes spinner2{0% {transform: rotate(0deg);}100% {transform: rotate(360deg);}}.spinner-3 div{animation:spinner2 1.2s cubic-bezier(0.5, 0, 0.5, 1) infinite;left:-32px;position:relative;top:-32px;transform-origin:32px 32px}.spinner-3 div::after{background-color:#fff;border-radius:50%;content:' ';height:6px;margin:-3px 0 0 -3px;position:absolute;width:6px}.spinner-3 div:nth-child(1){animation-delay:-.036s}.spinner-3 div:nth-child(1)::after{left:50px}.spinner-3 div:nth-child(2){animation-delay:-.072s}.spinner-3 div:nth-child(2)::after{left:45px}.spinner-3 div:nth-child(3){animation-delay:-.108s}.spinner-3 div:nth-child(3)::after{left:40px}.spinner-3 div:nth-child(4){animation-delay:-.144s}.spinner-3 div:nth-child(4)::after{left:35px}.spinner-3 div:nth-child(5){animation-delay:-.18s}.spinner-3 div:nth-child(5)::after{left:30px}.spinner-3 div:nth-child(6){animation-delay:-.216s}.spinner-3 div:nth-child(6)::after{left:25px}.spinner-3 div:nth-child(7){animation-delay:-.252s}.spinner-3 div:nth-child(7)::after{left:20px}.spinner-3 div:nth-child(8){animation-delay:-.288s}.spinner-3 div:nth-child(8)::after{left:15px}.spinner-3 div:nth-child(1)::after{top:50px}.spinner-3 div:nth-child(2)::after{top:54px}.spinner-3 div:nth-child(3)::after{top:57px}.spinner-3 div:nth-child(4)::after{top:58px}.spinner-3 div:nth-child(5)::after{top:57px}.spinner-3 div:nth-child(6)::after{top:54px}.spinner-3 div:nth-child(7)::after{top:50px}.spinner-3 div:nth-child(8)::after{top:45px}`
```

Comment:
wtf

Reply:
Mocked components to show some examples when not logged in 😄